services/shhext/service.go

```diff
@@ -251,3 +276,323 @@ func (s *Service) Stop() error {
 	s.tracker.Stop()
 	return nil
 }
+
+// JoinPublicChats join a set of public chats deriving the key from the chat name
+func (s *Service) JoinPublicChats(chats []string) ([]string, error) {
+
+	var response []string
+	if s.loadedFilters == nil {
+		s.loadedFilters = make(map[string]LoadedFilter)
+	}
+
+	for _, chatID := range chats {
```

Comment:
It looks like we can create multiple filters for the same chats. Is that desired?